astropy/time/formats.py

```diff
@@ -438,7 +462,7 @@ def set_jds(self, val1, val2):
             raise ScaleValueError("Cannot convert from '{}' epoch scale '{}'"
                                   "to specified scale '{}', got error:\n{}"
                                   .format(self.name, self.epoch_scale,
-                                          self.scale, err))
+                                          self.scale, err)) from err
```

Comment:
I didn't know about this syntax, but I like it!

Reply:
It's new. The tracebacks can be long but they can be lifesavers especially when the problem shows up in CI. I haven't systematically gone through and applied it everywhere, but I couldn't resist adding it when I saw an opportunity.

Not relevant to astropy but six has a function raise_from to let you do this even on python2.